### DISCUSSION TITLE
GoodbyeDPI UI v1.2.4

### DISCUSSION THREAD

**Discussion by Storik4pro** [Storik4pro/goodbyeDPI-UI#51] [Announcements]
## Что нового в этом обновлении?

- ✨ Теперь настройка уведомлений о наличии новой версии находиться в меню `"Система"`, вместо меню `"Обновление"`
- ✨ Теперь приложение добавляется в автозапуск только для текущего пользователя (Ранее добавлялось для всех)

## Список исправлений

- Была исправлена проблема, из-за которой пользовательские списки сайтов не сохранялись при обновлении компонента (#44)
- Теперь интерфейс параметров zapret отображается правильно вне зависимости от размера окна (#48)

## БЕТА - версии функций

- Была добавлена возможность быстро изменять списки сайтов в GoodCheck (#50)

## What's new in this update?

- ✨ The notification setting for new version availability has been moved to the `"System"` menu, instead of the `"Update"` menu.
- ✨ The application now adds itself to autostart only for the current user (previously it was added for all users).

## List of fixes

- Fixed an issue where custom site lists were not saved when updating a component (#44).
- The zapret settings interface now displays correctly regardless of the window size (#48).

## BETA features

- Added the ability to quickly edit site lists in GoodCheck (#50).

<hr /><em>This discussion was created from the release <a href='https://github.com/Storik4pro/goodbyeDPI-UI/releases/tag/1.2.4'>GoodbyeDPI UI v1.2.4</a>.</em>

**Comment by Nichohosobi** [comment]
На портативную антивир виндовса ругается и не качает...

**Reply by Storik4pro** [reply]
Попробуйте выключить SmartScreen защитника Windows на время скачивания программы

**Reply by BonePolk** [reply]
Ещё и на установщик начал ругаться

**Comment by LordArrin** [comment]
Было бы здорово, если бы можно было изменять размеры окна GoodCheck. На мониторах с высоким разрешением вдвойне больно.

**Reply by Storik4pro** [reply]
Вы имеете в виду окно настройки запуска или консоль работы компонента?

**Reply by LordArrin** [reply]
Окно работы GoodCheck, которое появляется при нажатии "Начать подбор"

**Reply by Storik4pro** [reply]
Было добавлено [в обновлении 1.2.5](https://github.com/Storik4pro/goodbyeDPI-UI/releases/tag/1.2.5) (#59)

**Comment by MEGADED-GIF** [comment]
Вы герой, спасибо

**Comment by LordArrin** [comment]
![image](https://github.com/user-attachments/assets/10380d2e-b2ef-40fc-837e-a21cd70fce6a)
возникли странные артефакты в окне программы

**Reply by LordArrin** [reply]
артефакты накапливаются, когда открываешь и закрываешь окно goodcheck

**Reply by Storik4pro** [reply]
Для решение проблемы мне необходима следующая информация о системе:

- версия ОС (например, Windows 10 22H2)
- установлены ли последние версии драйверов для видеокарты

Так же опишите подробнее, что нужно сделать чтобы воспроизвести проблему:

- Как вы закрывали GoodCheck
- Сколько времени работал GoodCheck
- Не возникали ли ошибки во время работы GoodCheck

**Reply by LordArrin** [reply]
Win 11 26100.2605, драйвера Intel последние, работал много и несколько раз закрывался с ошибкой из-за моих ошибок) настраивал zapret чтобы включить на роутере

**Reply by Storik4pro** [reply]
Похоже, что эта проблема связана с графикой Intel. Её исправление займет неопределенное время. Пока что попробуйте выключить размытие Mica, возможно, после этого артефакты пропадут

**Reply by Storik4pro** [reply]
Вы можете следить за прогрессом исправления в созданной теме #60

**Comment by beargorilla** [comment]
Спасибо, всё работает после переключения конфигов

**Comment by Navysailor1305** [comment]
Что делать если происходит ошибка такого типа?
![Снимок экрана (9)](https://github.com/user-attachments/assets/f96bd6ed-1ab8-41e7-b8d3-54e187e37c93)

**Reply by Storik4pro** [reply]
Остановите службу windrivert (нажмите `"остановить службу"`) и запустите zapret снова
